src/components/Button/Button.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import './Button.scss';
+
+class Button extends React.Component {
+  render() {
+    const { text, onClick, style } = this.props;
+
+    return (
+      <button onClick={onClick} className={style ? 'btn colored' : 'btn'}>
+        {text}
+      </button>
```

Comment:
```suggestion
      <button onClick={onClick} className={`btn ${style ? 'colored' : ''}`}>
        {text}
      </button>
```
이런식으로 중복을 줄이는 코드를 활용해보세요 :)